pkg/compute/init.go

```diff
@@ -67,6 +68,7 @@ func NewInitCommand(parent common.Registerer, globals *config.Data) *InitCommand
 	var c InitCommand
 	c.Globals = globals
 	c.CmdClause = parent.Command("init", "Initialize a new Compute@Edge package locally")
+	c.CmdClause.Flag("service-id", "Existing service ID to initialize from").Short('s').StringVar(&c.serviceID)
```

Comment:
`Existing service ID to initialize from` reads ambiguously to me. Am I using this service as the basis for a new service? Perhaps "Existing service ID to use. By default, this command creates a new service."